pkgs/applications/networking/instant-messengers/hangups/default.nix

```diff
@@ -0,0 +1,88 @@
+{ pkgs, stdenv, python3 }:
+let 
+  packageOverrides = self: super:{
+    ConfigArgParse = python3.pkgs.buildPythonPackage rec {
+      src = python3.pkgs.fetchPypi {
+        pname = "ConfigArgParse";
+        version = "0.11.0";
+        sha256 = "12dcl0wdsjxgphxadyz9bdzbvlwfaqgvba9s59ghajw4yqiyi2kc";
+      };
+      name = "configargparse-0.11.0";
+    };
+    aiohttp = python3.pkgs.buildPythonPackage rec {
+      src = python3.pkgs.fetchPypi {
+        pname = "aiohttp";
+        version = "2.3.10";
+        sha256 = "0r0955ar0yiwbp3war35z8zncs01nq84wdwk0v3s8f547dcadpca";
+      };
+      propagatedBuildInputs = [
+        pythonPackages.async-timeout
+        pythonPackages.chardet
+        pythonPackages.multidict
+        pythonPackages.yarl
+        pythonPackages.idna-ssl
+      ];
+      doCheck = false; 
+      name = "aiohttp-2.3.10";
+    }; 
+    ReParser = python3.pkgs.buildPythonPackage rec {
```

Comment:
Please add this one to the global `pythonPackages` in a seperate commit.

Reply:
Should this be added to `python-packages` even though v3.0.1 of `aiohttp` is already there?

Reply:
No, for every package in `pythonPackages` that you need a different version of, you have to use
```
<package> = super.<package>.overridePythonAttrs (oldAttrs: {
  version = "x.y.z";
  src = oldAttrs.src.override {
    inherit version;
    sha256 = "...";
  };
});
```